### PR TITLE
[WIP] changes in the gulpfile for watch

### PR DESCRIPTION
**Pull request by benmoufm**
the watchify option has been changed to avoid the killing of the process whenever there's something add in the files

### CODE REVIEW

File: gulpFile.js [hunk 1]
```diff
@@ -57,7 +57,12 @@ gulp.task('browserify', function (done) {
 });
 
 gulp.task('watch', function () {
-    return bundle(true);
+    return gulp.watch('src/*.*', ['default']);
+});
+
+gulp.task('default', function () {
+    return gulp.src(paths.jsSourcePath)
+        .pipe(browserify({transform: 'reactify'}));
```
Comment (jpetitcolas) [inline gulpFile.js]: You don't use React on this project. No need to reactify it.

File: gulpFile.js [hunk 2]
```diff
@@ -57,7 +57,12 @@ gulp.task('browserify', function (done) {
 });
 
 gulp.task('watch', function () {
-    return bundle(true);
+    return gulp.watch('src/*.*', ['default']);
+});
+
+gulp.task('default', function () {
+    return gulp.src(paths.jsSourcePath)
```
Comment (jpetitcolas) [inline gulpFile.js]: What is `paths.jsSourcePath`?

**Comment by benmoufm** [comment]
Can I merge the pull request now ?

File: gulpFile.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ gulp.task('browserify', function (done) {
 });
 
 gulp.task('watch', function () {
-    return bundle(true);
+    return gulp.watch('./lib/*.*', ['browserify']);
```
Comment (jpetitcolas) [inline gulpFile.js]: You can now remove the `watching` parameter in the `bundle` function.

**Pull request merged**